templates/components/layout.hbs

```diff
@@ -42,5 +44,9 @@
         {{~> page}}
     </div>
     {{> components/footer}}
+    <script src="/static/scripts/highlight.pack.js"></script>
+    <script>
+        hljs.initHighlightingOnLoad();
+    </script>
```

Comment:
Can we avoid the inline script? I'm going to try adding a Content Security Policy soon and inline scripts are pretty bad for it.